src/libs/conduit/python/py_src/relay/io/silo/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Lawrence Livermore National Security, LLC and other Conduit
+# Project developers. See top-level LICENSE AND COPYRIGHT files for dates and
+# other details. No copyright assignment is required to contribute to Conduit.
+
+###############################################################################
+# file: __init__.py
+# Purpose: Main init for the conduit relay io module.
+###############################################################################
+
+# silo support is optional, so drive on if we can't import
+try:
+    from .conduit_relay_io_silo_python import *
+except:
+    pass
```

Comment:
@cyrush how do you feel about this bonus try-catch?

None of the other things (MPI, Web, etc.) have logic like this. But it makes the windows CI pass.

Reply:
this is the one we need, you can remove the try catch case for silo (above)